tools/rosbag/README.md

```diff
@@ -0,0 +1,27 @@
+# rosbag
+This is a custom rosbag package which provides all the feature of [original rosbag](http://wiki.ros.org/rosbag/Commandline) with custom freq recording of topics.
+
+## how to record custom rosbag
+```shell
+rosbag record --file-name record.yaml -a
+```
+> this only throttles down the topics mentioned in record.yaml, other topics will be recorded at their publish rate.
+### param 
+* --file-name: file which has topics for custom recording
+
+## To-Do
+* currently it does not detect topic to record from file (file is only used for freq control of specific topic which is being recorded)
+
+
+## example:- record.yaml
+```yaml
+# topic_name: freq(must be an integer)
+"/chatter": 2
+"/raw_odom": 1
+"/realsense_bottom/accel/sample": 1
```

Comment:
there is no specific need for it to be an integer, why not just make it float/double?

Reply:
I would also suggest have an option of `-1` and this can record the topic in its default rate, this way we can maintain oen yaml file and not list topics again as arguments passed to rosbag